addon/-private/radar/utils/scroll-handler.js

```diff
@@ -80,7 +80,7 @@ export class ScrollHandler {
         info.left = cachedLeft;
 
         if (topChanged || leftChanged) {
-          Promise.resolve(info)
```

Comment:
If you rebase this off the commit i just accepted I'll merge it, this changes to `run.join`